persistence/src/main/scala/hmda/persistence/processing/HmdaFileValidator.scala

```diff
@@ -75,7 +77,7 @@ class HmdaFileValidator(submissionId: String) extends HmdaPersistentActor with T
   override def receiveCommand: Receive = {
 
     case BeginValidation =>
-      val ctx = ValidationContext(None, None)
+      val ctx = ValidationContext(None, getFilingPeriod(submissionId))
```

Comment:
With #562 merged in you can just get this from, I believe, `submissionId.period`!